app_vue/src/pages/MapPage.vue

```diff
@@ -18,7 +19,7 @@
 
   // make api call to get sensors
   const getLatestReadings = async () => {
-    const response = await axios.get('http://localhost:8080/latest_readings');
+    const response = await axios.get(API_URL + '/latest_readings');
```

Comment:
perhaps we can rename this to something a little more descriptive for clarity? `ENV_API_BASE_URL` or `ENV_DOMAIN_URL`?